app/templates/settings/applications.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="row">
```

Comment:
We don't want the tabs on each and every sub-route. This needs to be common to all. See how it's done for `events/live`, `events/draft`, `events/import`